attributes/network/11_ledger_tokens.adoc

```diff
@@ -30,13 +30,27 @@ include::{cddl}/11_ledger_tokens.cddl[tag=update]
 
 One can update a token to not have an owner by giving ownership to `null` (NOT anonymous).
 
+=== Extended Information
+
+A token can have optional and separately defined extended information.
+This information is specified in separate files and can be extended in the future.
+
+Events in the event log do not contain extended information, as these can be large in size.
```

Comment:
I would rephrase. 

The extended information attribute indices are present in the event log, not the associated data.